src/index.jsx

```diff
@@ -35,4 +34,4 @@ App.subscribe(APP_ERROR, (error) => {
   ReactDOM.render(<ErrorPage message={error.message} />, document.getElementById('root'));
 });
 
-App.initialize({ messages: [appMessages, headerMessages], loggingService: NewRelicLoggingService });
```

Comment:
Note: Your description says to initialize Segment, and all I see is logging going away. :)  Maybe you can add a comment here to better describe what is going on?  Thanks.

Reply:
The new version of frontend-base now initializes segment - we just needed to update to it.  The latest version also allowed the loggingService parameter to be optional, so I just removed it opportunistically.  Apologies, that wasn't very clear!